pdc/apps/release/migrations/0005_auto_20160314_1532.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+def populate_baseproduct_release_type(apps, schema_editor):
+    release_type_model = apps.get_model('release', 'BaseProduct')
+    try:
+        release_type = release_type_model.objects.get(name="ga")
+    except release_type_model.DoesNotExist:
+        return
+    model = apps.get_model('release', 'BaseProduct')
+    for i in model.objects.all():
+        i.type = release_type
+        i.save()
+
+
+def create_new_release_id(apps, schema_editor):
+    model = apps.get_model('release', 'Release')
+    for i in model.objects.all():
+        # this triggers populate_release_id() on pre-save and agenerate new release_id
+        i.save()
```

Comment:
This is not correct: according to [the documentation](https://docs.djangoproject.com/en/1.9/topics/migrations/#historical-models), the migration does not have access to any custom methods, so it does not trigger regeneration of the field. It would be better to duplicate code from `get_release_id` method here to make sure the migration is always using that.